pkg/template/template.go

```diff
@@ -38,9 +39,21 @@ func New(w io.Writer, width int, colorEnabled bool) Template {
 		output:       w,
 		tp:           tableprinter.New(w, true, width),
 		width:        width,
+		funcs:        template.FuncMap{},
 	}
 }
 
+// Funcs adds the elements of the argument map to the template's function map.
+// It must be called before the template is parsed.
+// It is legal to overwrite elements of the map including default functions.
+// The return value is the template, so calls can be chained.
+func (t *Template) Funcs(funcMap map[string]interface{}) *Template {
```

Comment:
Nit: I liked `RegisterFuncs` since it was a verb and more indicative of what it does.

Reply:
Yeah I went back and forth on that as well. I decided to just copy `text/template` method names as other gophers might be more familiar with it.